src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/client/PrisonApiClient.kt

```diff
@@ -88,23 +64,7 @@ class PrisonApiClient(
       }
       .block(apiTimeout)
   }
-
-  fun getCellHistory(bookingId: Int): PrisonerCellHistoryDto? {
```

Comment:
This isn't used

Reply:
It was left in, a while ago because we were switch over to another approach, tidy up task did not happen